library/core/data_callback/src/main/java/org/quiltmc/qsl/data/callbacks/api/predicate/AlwaysPredicate.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2023 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.data.callbacks.api.predicate;
+
+import com.mojang.serialization.Codec;
+
+import net.minecraft.util.Identifier;
+
+/**
+ * A predicate that is always true.
+ */
+public final class AlwaysPredicate<T> implements CodecAwarePredicate<T> {
+
+	public static final Identifier ID = new Identifier("quilt", "always");
```

Comment:
The last nitpick that I have about these var names is that I think `CODEC_ID` would be better than `ID` here; Could you change those?

Reply:
These should all be fixed